ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -123,7 +123,7 @@ func (do *DEXOnboarding) Layout(gtx C) D {
 			// TODO: Add Step Select Server and Step Registration Fee
 			return do.Theme.List(do.scrollContainer).Layout(gtx, 1, func(gtx C, i int) D {
 				gtx.Constraints.Max = image.Point{
-					X: int(cryptomaterial.MaxWidth),
+					X: gtx.Dp(values.MarginPadding450),
```

Comment:
@dreacot, please can you retest on your device?

Reply:
fixed